verification/curator-service/ui/src/redux/filters/thunk.ts

```diff
@@ -0,0 +1,30 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const countryListJsonFile =
+    'https://covid-19-aggregates.s3.amazonaws.com/country/latest.json';
+
+type countryObject = {
+    _id: string
+}
+
+export const fetchCountries = createAsyncThunk(
+    'countryList/fetchCountries',
```

Comment:
Here you can change `countryList/fetchCountries` to `filters/fetchCountries` to make things consistent